src/internal/pachd/builder.go

```diff
@@ -269,6 +271,19 @@ func (b *builder) registerPFSServer(ctx context.Context) error {
 	return nil
 }
 
+func (b *builder) registerStorageServer(ctx context.Context) error {
```

Comment:
Are we planning on putting more services within the Storage server? If not, I think it would be better to call this the file set server as that would keep the client and server name consistent.

Reply:
yeah, eventually. The storage server will eventually implement multiple services. We may eventually move the services to lower level servers though (fileset, chunk, etc.), but that would just be different servers at that point.